WebARonARKit/WebARonARKit.js

```diff
@@ -562,6 +565,35 @@
       return Array.from(this.planes_.values());
     };
 
+    this.createAnchor = function(modelMatrix) {
+      // Create the js anchor.
+      var anchor = new VRAnchor(modelMatrix);
+      var modelMatrixString = "";
+      for (var i = 0; i < 16; i++) {
+        modelMatrixString += modelMatrix[i] + (i < 15 ? "," : "");
+      }
+      window.webkit.messageHandlers.WebARonARKit.postMessage(
```

Comment:
why not just use this.dispatchEvent?

Reply:
Do not fully understand. Creating anchors is something the developers does manually whenever he/she wants to let the system know an anchor should be created. This portMessage is to let the native side know that an anchor should be added to the AR session.

Reply:
ah, I see. Sg